appcli/models/configuration.py

```diff
@@ -62,15 +62,23 @@ class Configuration(NamedTuple):
     templates to generate the final configuration files.
     """
 
-    seed_templates_dir: Path
+    baseline_templates_dir: Path
     """
-    Seed directory containing jinja2 templates used to generate the final
-    configuration files.
+    Directory containing the baseline set of jinja2 templates used to generate the final
+    configuration files. These template files are expected to remain static and should
+    only be overridden as a hotfix.
     """
 
     orchestrator: Orchestrator
     """ Orchestrator to use to launch Docker containers. """
 
+    configurable_templates_dir: Path = None
```

Comment:
I'd move this above `orchestrator`. It's closer related to `baseline_templates_dir` so is sensible to be near it

Reply:
In NamedTuples, `Fields with a default value must come after any fields without a default.`:
https://docs.python.org/3/library/typing.html#typing.NamedTuple